src/index.html

```diff
@@ -9,15 +9,17 @@
       html {
         --color-text-primary: #2a3443;
         --color-text-secondary: #5e6673;
-        --color-background: #db960a;
+        --color-background: #221f18;
```

Comment:
With this change (which is good!) the sidebar menu toggle is no longer visible. So I made that bn-dark-red and will push that to this branch.